.github/workflows/idol-notifications.yml

```diff
@@ -48,11 +52,38 @@ jobs:
           restore-keys: yarn-
       - name: Install
         run: yarn
-      - name: Profile update notifications
+      - name: TEC request notification
         env:
           FIREBASE_PRIVATE_KEY: ${{ secrets.FIREBASE_PRIVATE_KEY }}
           FIREBASE_PRIVATE_KEY_ID: ${{ secrets.FIREBASE_PRIVATE_KEY_ID }}
           OAUTH_CLIENTID: ${{ secrets.OAUTH_CLIENTID}}
           OAUTH_CLIENT_SECRET: ${{ secrets.OAUTH_CLIENT_SECRET }}
           OAUTH_REFRESH_TOKEN: ${{ secrets.OAUTH_REFRESH_TOKEN }}
         run: yarn workspace backend send-tec-notifications
+
+member-archive-reminder:
+  if: contains(github.event.schedule, '0 14 20 1,8 *')
```

Comment:
I think these two lines should be indented one more, so that `member-archive-reminder` is in `jobs`, and `if` is on the same indentation with `runs-on` like the other two jobs.

Reply:
I think is a symptom of not having formatting on the YAML files. We shouldn't have to point these styling issues out :(